sunpy/map/tests/test_mapbase.py

```diff
@@ -1181,6 +1181,21 @@ def test_quicklook(aia171_test_map):
         assert aia171_test_map._repr_html_() in html_string
 
 
+# requires dask array to run properly
+def test_dask_array(aia171_test_map):
+    dask_array = pytest.importorskip('dask.array')
+
+    da = dask_array.from_array(aia171_test_map.data, chunks=(1, 1))
+    pair_map = sunpy.map.Map(da, aia171_test_map.meta)
+
+    # Check that _repr_html_ functions for a dask array
+    html_dask_repr = pair_map._repr_html_()
+
```

Comment:
```suggestion
```
I don't think we need this line?

Reply:
oops that was leftover. Thanks for the catch